provision/aws/rosa_efs_create.sh

```diff
@@ -21,7 +21,18 @@ AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
 
 cd efs
 
-oc create -f aws-efs-csi-driver-operator.yaml
+
+echo "Installing Openshift cluster CSI drivers."  
+cat <<EOF | oc create -f -
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  generateName: openshift-cluster-csi-drivers-
+  namespace: openshift-cluster-csi-drivers
+EOF
+
+echo "Installing EFS CSI driver operator."
+oc apply -f aws-efs-csi-driver-operator.yaml
```

Comment:
If we define the OperatorGroup with `metadata.name` instead of `metadata.generateName` in `provision/aws/efs/aws-efs-csi-driver-operator.yaml`,  we should just be able to call `oc apply` here and remove the `oc create` call.